app/liquid_tags/golang_tag.rb

```diff
@@ -0,0 +1,30 @@
+class GolangTag < LiquidTagBase
+  PARTIAL = "liquids/golang".freeze
+  URL_REGEXP = %r{\Ahttps?://play.golang.org/p/[a-zA-Z0-9\-/]+\z}.freeze
+
+  def initialize(_tag_name, link, _parse_context)
+    super
+    stripped_link = ActionController::Base.helpers.strip_tags(link)
+    the_link = stripped_link.split(" ").first
+    raise "Invalid URL" valid_link?(the_link)
```

Comment:
Sorry, this was my mistake, I had a typo in the suggestion, this is missing the keyword 🤦 

```suggestion
    raise "Invalid URL" unless valid_link?(the_link)
```